plugins/line-numbers/prism-line-numbers.js

```diff
@@ -110,5 +116,29 @@
 
 		_resizeElement(pre);
 	});
+	
+	/**
+	 * Global exports
+	 */
+	Prism.plugins.lineNumbers = {
+		/**
+		 * Get node for provided line number
+		 * @param {Element} element pre element
+		 * @param {Number} number line number
+		 * @return {Element|undefined}
+		 */
+		getLine: function (element, number) {
+			if (element.tagName !== 'PRE' || !element.classList.contains(PLUGIN_CLASS)) {
+				return;
+			}
+			
+			var lineNumberStart = element.getAttribute('data-start') || 0;
+			var lineNumberRows = element.querySelector('.line-numbers-rows');
+			
+			number = Math.abs(lineNumberStart) + number;
+			
+			return lineNumberRows.children[number];
```

Comment:
@VitaliyR I think there is something wrong here. `children` is zero-indexed, while `number` (a value from `data-line`) is one-indexed. I tested in local and the highlighted line in indeed shifted down by one when line-numbers is used.